docs/src/main/asciidoc/writing-native-applications-tips.adoc

```diff
@@ -0,0 +1,129 @@
+include::./attributes.adoc[]
+= {project-name} - Tips for writing native applications
+
+This guide contains various tips and tricks for getting around various problems that might arise when attempting to run java apps as native images.
```

Comment:
s/apps/applications/
s/native images/native executables/ (Clément changed all the existing guides to use that)

Reply:
Cool, I will change all the usages here as well